locales/de.json

```diff
@@ -1113,6 +1113,8 @@
 		"link": {
 			"openInNewTab": "in neuem Tab öffnen"
 		},
+		"mobileNavMenu": "mobile nav toggle",
+		"openInNewTabWarning": "Opens in new tab",
```

Comment:
@luizzaly can you please provide a proper German translation here? Thanks.

Reply:
mobileNavMenu: "Mobiles Navigations Menu"
openInNewTabWarning:"Wird in einem neuem Tab geöffnet"